.flowconfig

```diff
@@ -2,21 +2,11 @@
 0.61.0
 
 [ignore]
-.*test
-./build
+.*/node_modules/jsxhint/.*
 
 [include]
-./lib
-index.js
 
 [libs]
 interfaces/
 
 [options]
-suppress_comment=\\(.\\|\n\\)*\\s*\\$FlowFixMe.*
-suppress_comment=\\(.\\|\n\\)*\\s*\\$FlowBug.*
-suppress_comment=\\(.\\|\n\\)*\\s*\\$FlowIgnore.*
-suppress_comment=\\(.\\|\n\\)*\\s*\\$FlowNewLine.*
-suppress_comment=\\(.\\|\n\\)*\\s*\\$FlowIssue
-esproposal.class_instance_fields=enable
```

Comment:
Guess we don't use these anymore? Nice to see.

Reply:
They are enabled by default :) But I have a plan for them. Binding arrays with method names is a bit noisy. Keeping arrows in constructor is not a big deal.